participants/coinsummer-tv.md

```diff
@@ -0,0 +1,53 @@
+# <Project Name>
+
+## (0) Project Name and Team Slack Handles
+Our Project Name is CoinSummerTV.
+Our Team Slack Handles is UNTPFH4HW, and with Display name Max.
+
+## (1) Brief Project Description
+[CoinSummerTV](https://coinsummer.tv) is a video site focus on cryptocurrency industry analysis, regulatory, trend prediction, trading strategy, fundamental analysis, industry technical analysis video, real-time tracking of the latest developments of crypto currency, and so on. We are planning to publish these videos to a decentralized network built on Filecoin and IPFS.
+
+## (2) Link to Project UI
+- [CoinSummerTV](https://coinsummer.tv)
+
+## (3) What does your application/UI do?
+Users can view, search, share all the videos on CoinSummerTV, and we are planning to support users upload their own videos. Authenticated users can also give a video some small tips using Lightning Network payment.
+
+## (4) If your project is using a curated dataset, which one are you using?
+This project will not use a curated dataset. All the videos on CoinSummerTV stored on our AWS S3 bucket. We are planning to store these videos to Filecoin and IPFS.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+All the videos on CoinSummerTV are MP4 format with three resolutions, 360P, 540P and 1080P. We have translated these videos, and add Chinese subtitles for them, we have exclusive rights to these data. These videos license available online, and don't contain any sensitive or illegal information.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We are planning to store at least 10TB videos to the Filecoin network, maybe more than 20TB if our users can upload more.
+
+## (7) How are you structuring the data?
+These videos are structured data stored to PostgreSQL database.
```

Comment:
Are any files larger than 4GB - there are currently technical limitations to files larger than 4GB.  If files are larger, what's the strategy to split and store on Filecoin and also retrieve and reconstruct these large files?

Reply:
All of these videos are 20MB to 100MB, and we are planning to send each video as a storage deal, so there will be no file larger than 4GB.